fix: wrong `&` explanation in nesting css selector. (#32280)
mdn/content: files/en-us/web/css/css_nesting/using_css_nesting/index.md

}
```

> In the following CSS we are creating the styles for `.card`, `.card h2` and then in the `h2` styles we nest the `.featured` class with the `&` nesting selector appended which creates a style for `.card :is(.featured h2)`.

Reviewer: your edit is correct, but i think we need to explain it. We can either edit the original from 

> which creates a style for `.card.featured h2`.

to

> which creates a style with a similar effect as `.card.featured h2`.

or we can use your edit, but we need to explain how the space between the `.card` and the `:is` is handled. That should be done here, then it will be understood on line 377. My interpretation is that it would actually be the equivalent of:
 
>`:is(.card h2):is(.featured h2)`

which it also is. I am wondering if it's easier to use your example, and explain how the space before the `:is()` works or my interpretation and explain how that one works.

Author: I agree with you that we need to further clarify how `.card :is(.featured h2)` works. I have added an explanatory sentence to clarify we can transform `.card :is(.featured h2)` to `:is(.card h2):is(.featured h2)` to better help us understand.

> In the following CSS we are creating the styles for `.card`, `.card h2` and then in the `h2` styles we nest the `.featured` class with the `&` nesting selector appended which creates a style for `.card :is(.featured h2)`. And we can furthur transform `.card :is(.featured h2)` to `:is(.card h2):is(.featured h2)`) to better help us understand.

Reviewer: ```suggestion
In the following CSS, we are creating the styles for `.card` and `.card h2`. Then, in the `h2` style block, we nest the `.featured` class with the `&` nesting selector appended which creates a style for `.card :is(.featured h2)`, which is the equivalent to `:is(.card h2):is(.featured h2)`.
```
